README.md

```diff
@@ -9,6 +9,12 @@ The main purpose of streamlink is to convert CPU-heavy flash plugins to a less C
 
 Streamlink is a fork of the [livestreamer](https://github.com/chrippa/livestreamer) project.
 
+Please note that by using this application you're bypassing any ads run by
+Twitch (adblock users do as well and
```

Comment:
Perhaps saying something along the lines of `...bypassing any ads ran by plugins such as Twitch`. So it still shows that we remain plugin-agnostic (no favouritism!) xD